distributed/dashboard/components/shared.py

```diff
@@ -454,11 +454,14 @@ def trigger_update(self):
 
 
 class SystemMonitor(DashboardComponent):
-    def __init__(self, worker, height=150, **kwargs):
+    def __init__(self, worker, height=150, last=None, **kwargs):
         self.worker = worker
```

Comment:
Given that this is a reference to the scheduler in this case (not a worker, we should probably fix the name) could we not just grab `self.worker.monitor.count` here instead of having to have a client and cluster object?

Reply:
Since this `SystemMonitor` is being created after the operation being profiled has finished, I think using `self.worker.monitor.count` here would end up setting the start time of the system plot to after the operation has finished?

My idea was that we want to capture what the value of `self.worker.monitor.count` would have been around the time `performance_report.__aenter__()` is called, since this is also where the start time is set.

Reply:
Ah right I see. Yeah that makes sense.